### PR TITLE
PKG-2341

### PR DESCRIPTION
**Pull request by cbouss**
Changes:
- increase build number
- relax host pins

Note: I won't upload this build. This is in prevision of the 3.12 buildout.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -23,13 +23,13 @@ outputs:
...
+        - packaging >=20.0
+        - typing-extensions
       run:
         - python
         - importlib-metadata  # [py<38]
         - packaging >=20.0
-        - setuptools >=45
+        - setuptools
         - tomli >=1.0.0  # [py<311]
         - typing-extensions
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:34]:
```suggestion
        - typing-extensions  # [py<311]
```

**Review by JeanChristopheMorinPerso — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml [hunk 2]
```diff
@@ -23,13 +23,13 @@ outputs:
         - pip
         - setuptools
         - wheel
-        - packaging 20.0
-        - typing-extensions 4.4.0
```
[the remark this answers is not in the document]
Comment (cbouss) [reply recipe/meta.yaml:27]: 7.1.0 doesn't have that restriction yet, so I left it:
https://github.com/pypa/setuptools_scm/blob/v7.1.0/pyproject.toml#L5

File: recipe/meta.yaml [hunk 3]
```diff
@@ -23,13 +23,13 @@ outputs:
...
+        - packaging >=20.0
+        - typing-extensions
       run:
         - python
         - importlib-metadata  # [py<38]
```
[the remark this answers is not in the document]
Comment (cbouss) [reply recipe/meta.yaml:30]: 7.1.0 doesn't list this restriction either:
https://github.com/pypa/setuptools_scm/blob/v7.1.0/setup.cfg#L31

File: recipe/meta.yaml (hunk 1 above)
Comment (cbouss) [reply recipe/meta.yaml:34]: or this one https://github.com/pypa/setuptools_scm/blob/v7.1.0/setup.cfg#L33

File: recipe/meta.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (JeanChristopheMorinPerso) [reply recipe/meta.yaml:27]: My bad.

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Pull request merged**